orcid-core/src/main/resources/org/orcid/core/template/digest_email_html.ftl

```diff
@@ -82,6 +82,7 @@
                     </tbody>					                      
 	        	</table>
 	        	</#list> 
+>>>>>>> master
```

Comment:
I changed file to make work with new pojo code. I think I merged in a
change from master as well
On 24 Feb 2016 02:11, "Robert Peters" <notifications@github.com> wrote:

> In
> orcid-core/src/main/resources/org/orcid/core/template/digest_email_html.ftl
> <https://github.com/ORCID/ORCID-Source/pull/2631#discussion_r53884598>:
>
> > @@ -82,6 +82,7 @@
> >                      </tbody>					
> >  	        	</table>
> >  	        	</#list>
> > +>>>>>>> master
>
> @TomDemeranville <https://github.com/TomDemeranville> how did this get in
> here :-) There shouldn't be changes in this section.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ORCID/ORCID-Source/pull/2631/files#r53884598>.
>